electron/webpack/webpack.config.renderer.ts

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+import webpack from 'webpack';
+import 'webpack-dev-server';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import DotEnv from 'dotenv-webpack';
+import TerserPlugin from 'terser-webpack-plugin';
+
+import paths, { resolveApp } from './paths';
+
+export const rendererConfig: webpack.Configuration = {
+  mode: 'development',
+  entry: paths.appIndexJs,
+  resolve: {
+    alias: {
+      './lib/platform': resolveApp('./src/lib/platform/index.desktop.ts'),
```

Comment:
Is there supposed to be an `index.desktop.ts` or is that auto-generated somewhere? I can't seem to locate it.

Reply:
Was a test on another branch - removed for now.